packages/falter-berlin-owmsh/Makefile

```diff
@@ -33,6 +33,14 @@ endef
 define Build/Compile
 endef
 
+define Package/falter-berlin-owmsh/postinst
+#!/bin/sh
+if [ -z $${IPKG_INSTROOT} ] ; then
+	( . /etc/uci-defaults/owm ) && rm -f /etc/uci-defaults/owm
+	rm -f /tmp/luci-indexcache
+fi
+endef
+
```

Comment:
I don't understand how this would install the cronjob. Can you please explain what it does?!?

Reply:
Thank you for the sharp review! You're right. I forgot to add the file, I'm sorry.

I've added the remaining thing in the updated version of this MR.

Reply:
do we really need to delete the luci-indexcache? from my understanding it is totally unrelated.